src/main/java/bio/terra/pdao/bigquery/WalkRelationship.java

```diff
@@ -16,6 +16,7 @@
     }
 
     private String[] columnNames;
+    private Boolean[] columnIsArrayOf;
```

Comment:
Is there a reason to use the boxed form instead of boolean? In general, we use boolean.

Reply:
No reason, will change